Robinhood/endpoints.py

```diff
@@ -1,5 +1,5 @@
 def login():
-    return "https://api.robinhood.com/api-token-auth/"
+    return "https://api.robinhood.com/oauth2/token/"
```

Comment:
IMO would be good to define something like https://github.com/Jamonek/Robinhood/pull/150/files#diff-b9ce533de4140ddc89a86a86547eb17bR1

Reply:
I completely agree! Thanks!